helix-view/src/lib.rs

```diff
@@ -15,6 +15,12 @@ pub mod view;
 #[derive(Clone, Copy, PartialEq, Eq, PartialOrd, Ord, Hash, Default, Debug)]
 pub struct DocumentId(usize);
 
+impl std::fmt::Display for DocumentId {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        f.write_fmt(format_args!("{}", self.0))
+    }
+}
```

Comment:
Why do we need to display `DocumentId`?

Reply:
We show it in the picker so you can look it up and specify the id on `:bclose`.

Reply:
Ah, I think this is better than debug then.